static/src/javascripts/bootstraps/common.js

```diff
@@ -54,6 +54,10 @@ define([
     'common/modules/ui/toggles',
     'common/modules/user-prefs',
     'common/modules/onward/breaking-news',
+
+    'text!common/views/international-message.html',
+    'text!common/views/international-control-message.html',
+
```

Comment:
Dunno. I was following the style in the file. But the style is dumb.

I'll get rid of it.